packages/backend/src/services/RenameService/rename.ts

```diff
@@ -616,6 +616,14 @@ export const getNameChanges = ({
     };
 };
 
+const addSuffixIfPrefix = (value: string | undefined, isPrefix: boolean) =>
+    value ? `${value}${isPrefix ? '_' : ''}` : '';
```

Comment:
how can value be undefined ? if undefined, this will return '', which means, it will match all models.